docs/tutorials/quickstart.md

```diff
@@ -4,10 +4,17 @@ Blueapi acts as a worker that can run bluesky plans against devices for a specif
 laboratory setup. It can control devices to collect data and export events to tell
 downstream services about the data it has collected.
 
-## Start ActiveMQ
+## Start RabbitMQ
 
-The worker requires a running instance of ActiveMQ, the simplest
-way to start it is to run it via a container:
+The worker requires a running instance of RabbitMQ. The easiest way to start it is
+ to execute the provided script:
+
+```
+    src/script/start_rabbitmq.sh
+```
+
+
+Alternatively, it is possible to run ActiveMQ via a container:
```

Comment:
Should: I couldn't get ActiveMQ to work (https://github.com/DiamondLightSource/blueapi/issues/694). Given this is a quickstart and it will be read by a new user I suggest we remove the option of ActiveMQ here and give the user the simplest configuration that works. If we want them to also be able to use ActiveMQ maybe we could put that in a new page just about that usecase?

Reply:
#694 is cause by the fact that at some point blueapi have stop setting default value [here](https://github.com/DiamondLightSource/blueapi/blob/627ffc3245ab85ccad838c968813f9023daa165d/src/blueapi/config.py#L86C13-L86C14)

Reply:
Ok, I still think that's it's confusing to a new user to be faced with both Active and Rabbit MQ but defer to @callumforrester @DiamondJoseph

Reply:
Yeah, I think it's fair enough to remove activemq. Long term I want to:
1. Make the CLI work without a message bus (albeit with less features)
2. Split the quickstart into just starting the server and running plan, and then a separate tutorial for connecting a message bus and getting rich feedback (I've actually already done the split in #674 so this should be rebased on top of that).

The idea is to make the quickstart as simple as possible.

Reply:
I've proposed adding a [RMQ Dockerfile](https://github.com/DiamondLightSource/blueapi/blob/026e3993a0d1d425177b55f9f43d6d828d7b0c52/tests/system_tests/Dockerfile) that installs the stomp plugin to the system_tests to make running those easier: this could be promoted out of that test dir and be a sanctified way of running a message bus for development?

Reply:
The CLI does (did?) work without the message bus, but without the ability to run plans (as we can't catch messages about the plan ending)

Reply:
I was [fairly against having our own Dockerfile](https://github.com/DiamondLightSource/blueapi/pull/668#discussion_r1822464761) (and therefore, one more setup step for the user). Is there no way of doing this with a vanilla image from Dockerhub and mounted config files?

Reply:
> The CLI does (did?) work without the message bus, but without the ability to run plans

As I said to @callumforrester in person. As a naïve user of `blueAPI` I feel like running plans is it's core function. To me "working without being able to run plans" is equivalent to not working.

Reply:
> I was [fairly against having our own Dockerfile](https://github.com/DiamondLightSource/blueapi/pull/668#discussion_r1822464761) (and therefore, one more setup step for the user). Is there no way of doing this with a vanilla image from Dockerhub and mounted config files?

I just about to remove the dockerfile